app/controllers/waste_carriers_engine/edit_forms_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module WasteCarriersEngine
+  class EditFormsController < FormsController
+    def new
+      super(EditForm, "edit_form")
+    end
+
+    # Override this method as user shouldn't be able to "submit" this page
+    def create; end
```

Comment:
I know we have this pattern existing already, but double questioning it now. Shouldn't we just not create the route?

Reply:
It's only temporary for this PR as we haven't built the rest of the journey yet. So maybe something to look into for future but I don't think it's a blocker here.